src/android/adb.ts

```diff
@@ -18,14 +19,24 @@ export function getAdbPath(customPath?: string): string {
     return path.resolve(customPath);
   }
 
-  return process.env.ANDROID_HOME
-    ? `${process.env.ANDROID_HOME}/platform-tools/adb`
-    : 'adb';
+  if (isWin) {
```

Comment:
How about using `path.join` to combine `process.env.ANDROID_HOME` , `platform-tools` and `adb`?? It should produce valid path for all the platforms, so this `if (isWin)` would not be needed.